win32ss/user/ntuser/window.c

```diff
@@ -1523,7 +1528,8 @@ NtUserBuildHwndList(
       {
          if(hDesktop == NULL && !(Desktop = IntGetActiveDesktop()))
          {
-            return ERROR_INVALID_HANDLE;
+            Status = ERROR_INVALID_HANDLE;
```

Comment:
Should we call the `SetLastNtError` in case of error for these? (basically all those you've modified).

Reply:
Moved `SetLastNtError` call to the end.